native/common/include/jp_platform_linux.h

```diff
@@ -41,9 +42,7 @@ class LinuxPlatformAdapter : public JPPlatformAdapter
 
 		if (jvmLibrary == NULL)
 		{
-			std::stringstream msg;
-			msg << "Unable to load DLL [" << path << "], error = " << dlerror();
-			JP_RAISE_RUNTIME_ERROR( msg.str().c_str());
+			JP_RAISE_OS_ERROR_UNIX( errno, path);
```

Comment:
on windows you have a human readable string, but it seems like on linux people just see an error code?

Reply:
I was sort of forced to do something non-standard on Windows because the Python version call for windows reporting is optional.   I can do the same  thing for linux if you would like.

Reply:
It looks like they should get symmetric errors

```
      case JPError::_os_error_unix:
      {
        PyObject* val = Py_BuildValue("(iz)", m_Error, (string("JVM DLL not found: ") + mesg).c_str());
        if (val != NULL)
        {
          PyObject* exc = PyObject_Call(PyExc_OSError, val, NULL);
          Py_DECREF(val);
          if (exc != NULL)
          {
            PyErr_SetObject(PyExc_OSError, exc);
            Py_DECREF(exc);
          }
        }
        return;
      }

      case JPError::_os_error_windows:
      {
        PyObject* val = Py_BuildValue("(izzi)", 2, (string("JVM DLL not found: ") + mesg).c_str(), NULL, m_Error);
        if (val != NULL)
        {
          PyObject* exc = PyObject_Call(PyExc_OSError, val, NULL);
          Py_DECREF(val);
          if (exc != NULL)
          {
            PyErr_SetObject(PyExc_OSError, exc);
            Py_DECREF(exc);
          }
        }
        return;
      }
```